docs/clusters/alpine/allocations.md

```diff
@@ -43,23 +43,25 @@ resources requested, job size, and QOS.
 ### Allocations
 When you receive a Research Computing account you are automatically 
 assigned a **Trailhead Auto-Allocation**, which grants you a fixed share 
-of `ucb-general`. The Trailhead is a great allocation for smaller jobs or 
+of `ucb-general`. RMACC users are granted a fixed share of `rmacc-general`. 
+The Trailhead is a great allocation for smaller jobs or 
 testing and benchmarking your code. However, if you consume more than your 
-fixed share of `ucb-general`,  your `LevelFS` will decrease, causing your 
+fixed share of `ucb-general` (or `rmacc-general`),  your `LevelFS` will decrease, causing your 
 priority score to decrease. The end result is that your jobs will sit 
-lower (read: longer) in the queue relative to other jobs. One way to 
+lower in the queue relative to other jobs. One way to 
 combat this is to apply for an allocation.
 
 ## Alpine Allocation Tiers
```

Comment:
Suggest changing this section title to Alpine Allocation Tiers for CU Boulder and RMACC Users